static/css/login_register_style.css

```diff
@@ -251,18 +298,16 @@ input[type=text]:placeholder {
   color: #0d0d0d;
 }
 
-.underlineHover:hover:after{
+.underlineHover:hover:after {
   width: 100%;
 }
 
-
-
 /* OTHERS */
 
 *:focus {
-    outline: none;
-} 
+  outline: none;
+}
 
 #icon {
-  width:60%;
+  width: 60%;
 }
```

Comment:
Just a regular CSS file used to style some of the containers and components in the website
no need to panic 🐼